src/applications/disability-benefits/all-claims/content/ratedDisabilities.jsx

```diff
@@ -8,12 +9,22 @@ import { NULL_CONDITION_STRING } from '../constants';
  * @property {String} diagnosticCode
  * @property {String} name
  * @property {String} ratingPercentage
+ * @property {String} maximumRatingPercentage
  * @param {Disability} disability
  */
-export const disabilityOption = ({ name, ratingPercentage, className }) => {
+export const disabilityOption = ({
+  name,
+  ratingPercentage,
+  className,
+  maximumRatingPercentage,
+}) => {
   // May need to throw an error to Sentry if any of these doesn't exist
   // A valid rated disability *can* have a rating percentage of 0%
   const showRatingPercentage = Number.isInteger(ratingPercentage);
+  const showMaxRatingMessage =
+    Number.isInteger(maximumRatingPercentage) &&
+    maximumRatingPercentage === ratingPercentage &&
+    String(localStorage.getItem('showDisability526MaximumRating')) === 'true';
```

Comment:
The feature flag toggles the education by passing the boolean value through localStorage. This was @christinec-fftc's recommendation, since this `disabilityOption` function for generating checkbox labels is so deeply nested away from the global redux state.